xcodeproject/xcscheme/xcscheme_test.go

```diff
@@ -52,157 +52,22 @@ func TestAppBuildActionEntry(t *testing.T) {
 	require.Equal(t, "ios-simple-objc", entry.BuildableReference.BlueprintName)
 	require.Equal(t, "container:ios-simple-objc.xcodeproj", entry.BuildableReference.ReferencedContainer)
 
+	pth, err := entry.BuildableReference.ReferencedContainerAbsPath("/project.xcodeproj")
+	require.NoError(t, err)
+	require.Equal(t, "/project.xcodeproj/ios-simple-objc.xcodeproj", pth)
+
 	require.True(t, entry.BuildableReference.IsAppReference())
 }
 
-func TestAppTestActionEntry(t *testing.T) {
-	var scheme Scheme
-	require.NoError(t, xml.Unmarshal([]byte(schemeContent), &scheme))
-
+func assertIosSimpleObjCSchemeTestActionEntry(scheme Scheme, t *testing.T) {
 	require.Equal(t, "Debug", scheme.TestAction.BuildConfiguration)
-	require.Equal(t, 2, len(scheme.TestAction.Testables))
+	require.Equal(t, 1, len(scheme.TestAction.Testables))
```

Comment:
On the [master branch](https://github.com/bitrise-io/sample-apps-ios-simple-objc/blob/master/ios-simple-objc/ios-simple-objc.xcodeproj/xcshareddata/xcschemes/ios-simple-objc.xcscheme#L44-L55) the scheme has 1 testable.